packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -45,8 +52,21 @@ export class ConnectorController {
      * @returns connector
      */
     getById = async (id: Id) => {
+        // add backwards compatibility code
         const res = await this.#editorAPI;
-        return res.getConnectorById(id).then((result) => getEditorResponseData<ConnectorInstance>(result));
+        return res
+            .getConnectorById(id)
+            .then((result) => getEditorResponseData<Next.ConnectorInstance>(result))
+            .then((resp) => {
+                const update: EditorResponse<ConnectorInstance> = { ...resp, parsedData: null };
+                if (resp.parsedData) {
+                    update.parsedData = this.#connectorCompatibilityTools.makeSingleConnectorBackwardsCompatible(
+                        resp.parsedData,
+                        this.#config.chiliEnvironmentUrl,

```

Comment:
I did a quick code search for this `chiliEnvironmentUrl` and found nothing using it. 

If this is merged, will it _**actually**_ work? 

Maybe @psamusev or @brapoprod knows.

Reply:
Moreover, `makeSingleConnectorBackwardsCompatible` scans this `chiliEnvironmentUrl` for a connector Id, which will most certainly not work in this case?

Reply:
> If this is merged, will it actually work?
We will check it on QA stage, before merge, to ensure